eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -252,6 +252,7 @@ jobs:
           displayName: "Execute Samples"
           env:
             TEST_MODE: "live"
+            AZURE_POD_IDENTITY_AUTHORITY_HOST: "FakeAuthorityHost" 
```

Comment:
Will this impact other tests? Like the identity tests? 

Can you add a comment here explaining why we are setting this?

Reply:
This change will only be for the "execute samples" task, so it will not impact other steps of the pipeline run and the test suite. I added the comment with more context